mailpoet/assets/js/src/listing/notices.jsx

```diff
@@ -29,6 +29,10 @@ function MailerError(props) {
   if (props.mta_log.error.operation === 'send' && props.mta_method === 'MailPoet' && MailPoet.hasInvalidMssApiKey) {
     return null;
   }
+  // do not display Email Volume Limit reached error twice
+  if (props.mta_method === 'MailPoet' && props.mta_log.error.operation === 'email_limit_reached') {
+    return null;
+  }
```

Comment:
After hiding this duplicate message is it still necessary to have it? I mean it is not quite complex code to [create it in the MSS Error mapper](https://github.com/mailpoet/mailpoet/pull/3989/commits/8465035f3ce58ba1cb46156ab3f6e3969d0f228e#diff-a6737f295bd861863026c7f41274b9818006adda1f7683f5c4e374d3d52ae8beR177-R196) and now it seems to me that we will hide it anyway. Or is there some case when it will be displayed?

Reply:
The error message is visible on the system status page. However, I'm not sure if it is necessary. What do you think @NeosinneR ?

Reply:
If we already show an error message when the subscriber limit is reached on the System Status page, I'd also show an error message when an email volume limit is reached. Because both error messages indicate that there is something wrong with your plan or sending, the System Status page seems like a place to look for the reason for such an error.

Reply:
Aha. I forgot about the status page. 👍  Lets keep the message then